hassio/src/dashboard/hassio-update.ts

```diff
@@ -167,6 +167,7 @@ export class HassioUpdate extends LitElement {
           font-size: var(--paper-font-subhead_-_font-size);
           font-weight: 500;
           margin-bottom: 0.5em;
+          color: var(--primary-text-color);
```

Comment:
Same should be done for the icon

Reply:
Done in cc2090a26cc58b52049cc01b8173a5a8024d2185.